src/current/cockroachcloud/folders.md

```diff
@@ -154,13 +150,6 @@ Your service account must have one of the following roles to read a folder's con
 - [Folder Admin]({% link cockroachcloud/authorization.md %}#folder-admin) or [Folder Mover]({% link cockroachcloud/authorization.md %}#folder-mover).
 - [Cluster Administrator]({% link cockroachcloud/authorization.md %}#cluster-administrator), [Cluster Developer]({%link cockroachcloud/authorization.md %}#cluster-developer), [Cluster Creator]({% link cockroachcloud/authorization.md %}#cluster-creator), or [Cluster Operator]({% link cockroachcloud/authorization.md %}#cluster-operator).
 
-{{site.data.alerts.callout_success}}
```

Comment:
is the stuff in here no longer true? 

or is this a style edit? 

is this already mentioned elsewhere?

Reply:
It's just no longer true / has been fixed.